reV/supply_curve/sc_aggregation.py

```diff
@@ -281,8 +285,42 @@ def __init__(self, excl_fpath, tm_dset, econ_fpath=None,
             ``layer_dset_name`` is a dataset in the exclusion h5 file
             and the ``kwarg: value`` pair is a keyword argument to
             the :class:`reV.supply_curve.exclusions.LayerMask` class.
-            If ``None`` or empty dictionary, no exclusions are applied.
-            By default, ``None``.
+            For example::
+
+                excl_dict = {
+                    "typical_exclusion": {
+                        "exclude_values": 1,
+                    },
+                    "another_exclusion": {
+                        "exclude_values": [2, 3],
+                        "weight": 0.5
+                    },
+                    "exclusion_with_nodata": {
+                        "exclude_range": [10, 100],
+                        "exclude_nodata": True,
+                        "nodata_value": -1
+                    },
+                    "partial_setback": {
+                        "use_as_weights": 1
```

Comment:
This should probably be a boolean True to avoid confusion with a weight multiplier of 1.

Reply:
Good catch; I updated to `True`